cmd/access/main.go

```diff
@@ -391,6 +395,37 @@ func main() {
 			}
 			return ping, nil
 		})
+
+		if nodeBuilder.ParticipatesInUnstakedNetwork() {
+			nodeBuilder.
+				Component("unstaked sync request proxy", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+					proxyEngine := splitter.New(node.Logger, engine.SyncCommittee)
+
+					// register the proxy engine with the unstaked network
+					unstakedNetworkConduit, err = anb.UnstakedNetwork.Register(engine.SyncCommittee, proxyEngine)
+					if err != nil {
+						return nil, fmt.Errorf("could not register unstaked sync request proxy: %w", err)
+					}
+
+					return proxyEngine, nil
+				}).
+				Component("unstaked sync request handler", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+					syncRequestHandler := synceng.NewRequestHandler(
+						node.Logger,         // TODO: use different logger for unstaked network?
+						node.Metrics.Engine, // TODO: use different metrics for unstaked network?
```

Comment:
@vishalchangrani probably we do right? Otherwise the metrics for the staked and unstaked sync request will get mixed together